packages/components/src/components/DataTableSkeleton/DataTableSkeleton.jsx

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2024 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { usePrefix } from '@carbon/react';
+import { classNames } from '@tektoncd/dashboard-utils';
+
+export default function DataTableSkeleton({
+  filters,
+  headers,
+  rowCount = 5,
+  columnCount = 5,
+  compact = false,
+  className,
+  showDescription,
+  showHeader = true,
+  showToolbar = true,
+  size,
+  ...rest
+}) {
+  const prefix = usePrefix();
+
+  const dataTableSkeletonClasses = classNames(className, {
+    [`${prefix}--skeleton`]: true,
+    [`${prefix}--data-table`]: true,
+    [`${prefix}--data-table--compact`]: compact,
+    [`${prefix}--data-table--${size}`]: true
+  });
+
+  const rowRepeat = rowCount;
```

Comment:
`rowRepeat` isn't changed so could just re-use `rowCount` on lines 47 and 49?

Reply:
yep good catch, did some refactoring before opening the PR and missed this on cleanup